packages/clerk-js/src/core/fapiClient.ts

```diff
@@ -197,8 +197,15 @@ export default function createFapiClient(clerkInstance: Omit<Clerk, 'proxyUrl'>)
     } catch (e) {
       clerkNetworkError(urlStr, e);
     }
+
+    const responseHeaders = {
+      country: response.headers.get('x-country'),
+    };
+
     const json: FapiResponseJSON<T> = await response.json();
-    const fapiResponse: FapiResponse<T> = Object.assign(response, { payload: json });
+    const fapiResponse: FapiResponse<T> = Object.assign(response, {
+      payload: { ...json, meta: { ...json?.meta, responseHeaders } },
```

Comment:
Why do we need `...json?.meta` instead of just  `...json.meta`?

Reply:
Also, the FapiResponseJSON needs to be updated with the responseHeaders key:

```
export interface FapiResponseJSON<T> {
  response: T;
  client?: ClientJSON;
  errors?: ClerkAPIErrorJSON[];
  meta?: {
    client?: ClientJSON;
    session_id?: string;
    responseHeaders?: { country: string }
  };
}
```

Reply:
@nikosdouvlis this is purely for safety. Do you think it is not needed?